README.md

```diff
@@ -1,48 +1,83 @@
 # scradd
-
+![](https://cdn.discordapp.com/icons/938438560925761619/d886f0b0867e67c2211b7086c0651590.webp?size=80)
 ## About
 
-This is a Discord bot for the [Scratch Addons Discord server](https://discord.gg/Cs25kzs889). It was partially made because it was the servers' one year anniversary, and partially because we wanted a bot that we had ultimate customization over.
+Scradd is the Discord bot for the [Scratch Addons Discord server](https://discord.gg/Cs25kzs889). It was partially made because servers' one year anniversary, and partially because we wanted a bot that we had ultimate customization over.
+
+Logo made by Weirdo#8115 (Discord) and @retronbv.
+
+Cloud-hosted on [opeNode.io](https://www.openode.io/).
+
+## Table of Contents
+ [Features](#features) | [Contributing](#contributing) 
+| --- | ---
+| [Potatoboard](#potatoboard) | [How to contribute](#how-to-contribute)
+| [Suggestions - /suggestion](#suggestions) | [Contributors](#contributors)
+| [Bug Reports - /bugreport](#bug-reports) |
+| [Addon Info - /addon](#addon-addon) |
 
-Logo made by @(does weirdo have a github) and @retronbv.
 
-Cloud hosted on [opeNode.io](https://www.openode.io/).
 
 ## Features
 
 ### Potatoboard
+  
+Messages with more than 8 :potato: reactions are shown in the #potatoboard channel. 
```

Comment:
it's 6 again

Reply:
bruh

Reply:
8 was too many and colander wanted it to be an even number